py/server/deephaven/server/plugin/object/__init__.py

```diff
@@ -4,12 +4,13 @@
 
```

Comment:
@jmao-denver should we also rename this module to object_type to avoid any import confusion?

Reply:
Yes.